src/tracking/trackers/tests/forgot-password.test.jsx

```diff
@@ -0,0 +1,37 @@
+import { createEventTracker, createPageEventTracker } from '../../../data/segment/utils';
+import {
+  categories,
+  eventNames,
+  forgotPasswordPageViewedEvent,
+  trackForgotPasswordPageEvent,
+} from '../forgotpassword';
+
+// Mock createEventTracker function
+jest.mock('../../../data/segment/utils', () => ({
+  createEventTracker: jest.fn().mockImplementation(() => jest.fn()),
+  createPageEventTracker: jest.fn().mockImplementation(() => jest.fn()),
+}));
+
+describe('Tracking Functions', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('trackForgotPasswordPageEvent function', () => {
```

Comment:
Please add some descriptive unit test names like:
```suggestion
  it('should fire trackForgotPasswordPageEvent', () => {
```

Reply:
resolved